packages/builder/src/components/backend/DataTable/modals/CreateEditColumn.svelte

```diff
@@ -191,8 +196,10 @@
     // don't make field IDs for auto types
     if (type === AUTO_TYPE || autocolumn) {
       return type.toUpperCase()
-    } else {
+    } else if (type === FieldType.BB_REFERENCE) {
```

Comment:
should we just check here if it has a `subtype` and do this? Just to cover all sub-type supporting columns?

Reply:
This was a 'hack' that was added for bb reference. This assumes that these columns' subtypes, and the old attachments have not

Reply:
Ah I see - makes sense!